File: Herencia/Clases/src/clases/Vehiculo.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package clases;

/**
 *
 * @author ESTUDIANTES
 */

public class Vehiculo {
    
    private String fechaFabricacion;
    private String VINChasis;
    private String VINMotor;

    public Vehiculo(String fechaFabricacion, String VINChasis, String VINMotor) {
        this.fechaFabricacion = fechaFabricacion;
        this.VINChasis = VINChasis;
        this.VINMotor = VINMotor;
    }

    public String getFechaFabricacion() {
        return fechaFabricacion;
    }

    public void setFechaFabricacion(String fechaFabricacion) {
        this.fechaFabricacion = fechaFabricacion;
    }

    public String getVINChasis() {
        return VINChasis;
    }

    public void setVINChasis(String VINChasis) {
        this.VINChasis = VINChasis;
    }

    public String getVINMotor() {
        return VINMotor;
    }

    public void setVINMotor(String VINMotor) {
        this.VINMotor = VINMotor;
    }
}

File: Clases python/Clases/Vehiculo.py
class Vehiculo:
    def __init__(self, fecha_fabricacion, VIN_Chasis, VINMotor):
        self.fecha_fabricacion = fecha_fabricacion
        self.VIN_Chasis = VIN_Chasis
        self.VINMotor = VINMotor

    @property
    def fecha_fabricacion(self):
        return self._fecha_fabricacion

    @fecha_fabricacion.setter
    def fecha_fabricacion(self, valor):
        self._fecha_fabricacion = valor
    

    @property
    def VIN_Chasis(self):
        return self._VIN_Chasis

    @VIN_Chasis.setter
    def VIN_Chasis(self, valor):
        self._VIN_Chasis = valor


    @property
    def VINMotor(self):
        return self._VINMotor

    @VINMotor.setter
    def VINMotor(self, valor):
        self._VINMotor = valor
    

